reverbePrintemps/spiced
# Update styling of social card to specifications

# Context

This PR aims to tackle [the social card challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/css-basics/challenges-css-basics.md#social-card) by updating the styles of the social card to [specifications](#specs)

# Specs

![social-card](https://github.com/reverbePrintemps/spiced/assets/59572064/4bf88156-f263-4b1a-a2cb-bfe0d7f4f090)

# Changes

- [x] Added classnames
- [x] Updated styles

# Test

1. Clone repo
2. Checkout branch
3. Run `index.html` in the browser
4. Witness updated styles
5. Optional: hover over them to confirm hover states

# TODO


# GIF

![](https://media.giphy.com/media/dGSaFEhCcojvbtrg2O/giphy.gif)
